.gitignore

```diff
@@ -12,3 +12,4 @@ build
 bin/
 .classpath
 .vscode
+sample-extension-plugin/src/test/resources/
```

Comment:
Is it accidental? Seems like legit non-ignorable target

Reply:
It's not. The old code was downloading older versions of job-scheduler into `src/test/resources/bwc/job-scheduler` so I kept it. The folder doesn't exist in the tree. Maybe it should download to /tmp, but I didn't want to change too many things. WDYT?

Reply:
Oh ... I see, thanks for explaining, let's keep it like that indeed